### PR TITLE
Use TwoPaneLayout in Companion Pane sample

### PR DESCRIPTION
**Pull request by khalp**
Update the companion pane sample to use TwoPaneLayout

### CODE REVIEW

**Review by CesarValiente — APPROVED** [review]
LGTM! Thanks Kristen!

File: ComposeSamples/CompanionPane/src/main/java/com/microsoft/device/display/samples/companionpane/HomePage.kt [hunk 1]
```diff
@@ -93,97 +102,132 @@ fun SetupUI(windowInfoRep: WindowInfoRepository) {
...
+
+@Composable
+fun Pane2(screenState: ScreenState) {
+    when (screenState) {
+        ScreenState.DualPortrait -> {
+            ShowWithTopBar(
+                content = { DualPortraitPane2() }
+            )
         }
+        ScreenState.DualLandscape -> DualLandscapePane2(Modifier.background(color = MaterialTheme.colors.background))
```
Comment (joyl1216) [inline ComposeSamples/CompanionPane/src/main/java/com/microsoft/device/display/samples/companionpane/HomePage.kt:156]: I think you can just set `Modifier.background(color = MaterialTheme.colors.background` to `Column` inside `DualLandscapePane2`. No need to abstract the `modifier` as the parameter.

File: ComposeSamples/CompanionPane/src/main/java/com/microsoft/device/display/samples/companionpane/HomePage.kt [hunk 2]
```diff
@@ -93,97 +102,132 @@ fun SetupUI(windowInfoRep: WindowInfoRepository) {
...
-        Box(
-            modifier = Modifier
-                .fillMaxSize()
-                .padding(start = 10.dp, end = 10.dp, top = 30.dp, bottom = 30.dp)
-                .clipToBounds()
-                .weight(.9f),
-            contentAlignment = Alignment.Center
-        ) {
-            ImagePanel(modifier = Modifier.padding(start = 10.dp, end = 10.dp))
+fun DualPortraitPane1(modifier: Modifier = Modifier) {
```
Comment (joyl1216) [inline ComposeSamples/CompanionPane/src/main/java/com/microsoft/device/display/samples/companionpane/HomePage.kt:162]: I think you can remove `modifier: Modifier = Modifier`.

File: ComposeSamples/CompanionPane/src/main/java/com/microsoft/device/display/samples/companionpane/HomePage.kt [hunk 3]
```diff
@@ -93,97 +102,132 @@ fun SetupUI(windowInfoRep: WindowInfoRepository) {
...
+        modifier = modifier.fillMaxSize(),
+        verticalArrangement = Arrangement.SpaceEvenly
+    ) {
+        ImagePanel(Modifier.padding(horizontal = 30.dp))
+        EffectPanel()
+    }
+}
+
+@Composable
+fun DualPortraitPane2(modifier: Modifier = Modifier) {
```
Comment (joyl1216) [inline ComposeSamples/CompanionPane/src/main/java/com/microsoft/device/display/samples/companionpane/HomePage.kt:173]: I think you can remove `modifier: Modifier = Modifier` here too.

File: ComposeSamples/CompanionPane/src/main/java/com/microsoft/device/display/samples/companionpane/HomePage.kt [hunk 4]
```diff
@@ -93,97 +102,132 @@ fun SetupUI(windowInfoRep: WindowInfoRepository) {
...
+            VignettePanel(modifier = Modifier.width(longSlideWidth))
+            BrightnessPanel(modifier = Modifier.width(longSlideWidth))
         }
-        Column(
+        ShortFilterControl()
+    }
+}
+
+@Composable
+fun DualLandscapePane1(modifier: Modifier = Modifier) {
```
Comment (joyl1216) [inline ComposeSamples/CompanionPane/src/main/java/com/microsoft/device/display/samples/companionpane/HomePage.kt:190]: Same here. :)

File: ComposeSamples/CompanionPane/src/main/java/com/microsoft/device/display/samples/companionpane/HomePage.kt [hunk 5]
```diff
@@ -93,97 +102,132 @@ fun SetupUI(windowInfoRep: WindowInfoRepository) {
...
+            .padding(bottom = 20.dp)
+            .clipToBounds(),
+        contentAlignment = Alignment.Center
+    ) {
+        ImagePanel(modifier = Modifier.padding(all = 20.dp))
+    }
+}
+
+@Composable
+fun DualLandscapePane2(modifier: Modifier = Modifier) {
```
Comment (joyl1216) [inline ComposeSamples/CompanionPane/src/main/java/com/microsoft/device/display/samples/companionpane/HomePage.kt:203]: Same here. :)

**Review by joyl1216 — APPROVED** [review]

**Pull request merged**